### ISSUE TITLE
Product Services

### ISSUE THREAD

**Issue by sametakbal** [sametakbal/kips-shop#2]
Product CRUD operations

**Comment by Eyablg-ops** [comment]
Hello @sametakbal is this issue still open for contribution? If no work is currently in progress, I'd love to take it on!

**Comment by sametakbal** [comment]
Of course, it is open to improvement; I would be very happy about that. @Eyablg-ops

**Comment by Eyablg-ops** [comment]
@sametakbal I've submitted a pull request. Could you please review it when you have a chance?
